index.html

```diff
@@ -204,11 +204,60 @@ <h1>Multi-Post Stories</h1>
               Bootstrap
               <span> <img src="images/image3.png" alt="bootstrap" /></span>
             </li>
-            <li>Ruby</li>
+            <li class="list-sumbua">Ruby</li>
           </ul>
           <button class="seeproject-btn" type="submit">See Project</button>
         </div>
       </section>
+      <form action="https://formspree.io/f/xqkngpry" method="post">
+        <h1>CONTACT US</h1>
+        <p>
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+        <input
+          class="input-area"
+          type="text"
+          placeholder="Enter your name"
+          maxlength="30"
+          name="name"
+          required
+        />
+        <input
+          class="input-area mail"
+          type="email"
+          placeholder="Enter your email"
+          name="email"
+          required
+        />
+        <textarea
+          name="textarea"
+          class="text-area"
+          maxlength="500"
+          placeholder="Enter text here"
+          required
+        ></textarea>
```

Comment:
- [ ] According to your Figma template, you need to add padding in your `input` fields and `textarea`. 

Figma template:
![image](https://user-images.githubusercontent.com/52635378/165395057-52703ecc-feba-4ffb-b575-f2efdbb8b41b.png)

Your project looks like this:
![image](https://user-images.githubusercontent.com/52635378/165395139-e9c4684f-60cc-48a0-b034-1c477998daf4.png)